src/picterra/client.py

```diff
@@ -146,8 +158,13 @@ def upload_raster(self, filename, name, folder_id=None, captured_at=None):
         self._wait_until_operation_completes(resp.json())
         return raster_id
 
-    def list_rasters(self):
+    def list_rasters(self, folder_id: str = ''):
```

Comment:
In Python, it's better to use None to indicate a parameter is not set. So I'd have `folder_id=None` here to indicate we don't want filtering

Reply:
Checked